src/pages/EnablePayments/AdditionalDetailsStep.js

```diff
@@ -292,91 +221,62 @@ class AdditionalDetailsStep extends React.Component {
                             {this.props.translate('additionalDetailsStep.helpLink')}
                         </TextLink>
                     </View>
-                    <FormScrollView ref={el => this.form = el}>
+                    <Form
+                        formID={ONYXKEYS.WALLET_ADDITIONAL_DETAILS}
+                        validate={this.validate}
+                        onSubmit={this.activateWallet}
+                        submitButtonText={this.props.translate('common.saveAndContinue')}
+                        style={[styles.mh5, styles.flexGrow1]}
+                    >
                         <View style={[styles.mh5, styles.mb5]}>
                             <View style={styles.mt4}>
```

Comment:
Agree with @NikkiWines about the horizontal spacing - I also don't really see the need for top & bottom spacing, maybe we can keep it though since it existed before? I tried removing both of these `View`s and the Form looked pretty nice

```suggestion
                        style={[styles.mh5, styles.flexGrow1]}
                    >
```

Reply:
Done